smurfs/src/ReadMeAnswers.md

```diff
@@ -0,0 +1,42 @@
+### Self-Study/Essay Questions
+
+### Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
+
+-----------------------------------------------------
+
+1)  In your own words, describe actions, reducers and the store and their role in Redux. What does each piece do? Why is the store known as a 'single source of truth' in a redux application?
+
+    a. `Actions` is defined as: a regular JavaScript object that usually has two properties, type and payload.
```

Comment:
Great answer, just remember that the type property is the one that is required by redux.